attributes/default.rb

```diff
@@ -43,6 +43,14 @@
 }
 default['consul']['source_revision'] = 'master'
 
+# GOMAXPROCS should be greater than 1, see
+# https://groups.google.com/forum/#!topic/consul-tool/qewFEqgAoF8
+if node['cpu']['total'] > 1
+  default['consul']['gomaxprocs'] = node['cpu']['total']
```

Comment:
I guess I am just not sold that this needs to be able to be tuned as a node attribute. Perhaps we can set it as a variable in the template instead of rendering the attribute? /cc @reset any thoughts?